libyul/backends/wasm/EWasmToText.cpp

```diff
@@ -0,0 +1,160 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/**
+ * Component that transforms interval EWasm representation to text.
+ */
+
+#include <libyul/backends/wasm/EWasmToText.h>
+
+#include <libdevcore/StringUtils.h>
+
+#include <boost/algorithm/string/join.hpp>
+#include <boost/algorithm/string/replace.hpp>
+#include <boost/range/adaptor/transformed.hpp>
+
+using namespace std;
+using namespace yul;
+
+string EWasmToText::run(
+	vector<wasm::GlobalVariableDeclaration> const& _globals,
+	vector<wasm::FunctionDefinition> const& _functions
+)
+{
+	string ret = "(module\n\n";
+	for (auto const& g: _globals)
+		ret += "    (global $" + g.variableName + " (mut i64) (i64.const 0))\n";
+	for (auto const& f: _functions)
+		ret += transform(f) + "\n";
+	return move(ret) + ")\n";
```

Comment:
Consider to add `std::`?

Reply:
We decided to only use `std::move` if there is a conflict with some other imported `move`.